src/dune_pkg/solver_env.ml

```diff
@@ -224,19 +224,26 @@ let default =
   { flags = Variable.Flag.Set.all
   ; sys = Variable.Sys.Bindings.empty
   ; const = Variable.Const.bindings
-  ; repos = [ Workspace.Repository.Name.of_string ":standard" ] (* TODO *)
+  ; repos = [ Workspace.Repository.Name.of_string ":standard" ]
   }
 ;;
 
+let repos_of_ordered_set ordered_set =
+  Dune_lang.Ordered_set_lang.eval
+    ordered_set
+    ~parse:(fun ~loc:_ -> Workspace.Repository.Name.of_string)
```

Comment:
I think you can use `parse_string_exn` here to make use of the location.

Reply:
Ah cool, that exists indeed. Bit of a pity it takes a tuple but then again the currying made it harder to read than it needed to.